src/lib.rs

```diff
@@ -4,493 +4,470 @@
 
 #![no_std]
 
-//! Provides a trait for numeric types to perform combined multiplication and division with
```

Comment:
Why are you removing all the docs and various other code?

Reply:
@sdroege 
The original code has been restored

Reply:
Thanks! But how did this happen? :)

Reply:
It's updated again to fix the clippy issue

Reply:
Hi @sdroege 
Could you add the Linked issue to this PR?

https://github.com/sdroege/rust-muldiv/issues/3
This one :)